website/docs/developer/development-guide/code/setup-dev-env.md

```diff
@@ -49,11 +49,15 @@ Lambdas should be built and tested within individual Virtual Environments.
 1. With the VEnv activated, install requirements.
    ```commandline
    pip install boto3
-   pip install -r requirements.txt
+   pip install -r requirements.txt --trusted-host pypi.org --trusted-host files.pythonhosted.org
    ```
 1. Install additional development requirements.
    ```commandline
    pip install coverage
    pip install pylint
-   pip install -r requirements-dev.txt
-   ```
\ No newline at end of file
+   pip install -r requirements-dev.txt --trusted-host pypi.org --trusted-host files.pythonhosted.org
```

Comment:
See note from above on flags. Wondering if we should remove coverage and pylint? or possibly put a note to add all of the dev libraries that should be used like black, isort, pylint, coverage, etc in a note that they should be installed with a link to our template for run_test.sh? Thoughts?

Reply:
`coverage` and `pylint` are mentioned in separate pages that focus on checking these things outside of our `run_tests.sh` script. If you want to refactor these pages, that is beyond the scope of the PR.